src/journal/journalctl.c

```diff
@@ -2064,6 +2064,9 @@ static int simple_varlink_call(const char *option, const char *method) {
 }
 
 static int flush_to_var(void) {
+        if (access("/run/systemd/journal/flushed", F_OK) >= 0)
```

Comment:
i new code i'd prefer if we'd log about unexpected errros from access(). i.e. please change to:

```c
if (access("…", F_OK) < 0) {
        if (errno != ENOENT)
                return log_error_errno(errno, "Unable to check for existance of …: %m");
} else
        return 0; /* Already flushed, no need to contact journald */
```

Reply:
fixed.